### PR TITLE
Bug 1617128 - Enable fennec-profile-manager project

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

File: src/scriptworker/constants.py [hunk 1]
```diff
@@ -361,16 +361,27 @@
                         {
                             "project:mobile:android-components:releng:beetmover:bucket:maven-production": "android-components-repo",
                             "project:mobile:android-components:releng:beetmover:bucket:maven-snapshot-production": "android-components-repo",
+
                             "project:mobile:fenix:releng:googleplay:product:fenix": "fenix-repo",
                             "project:mobile:fenix:releng:signing:cert:nightly-signing": "fenix-repo",
                             "project:mobile:fenix:releng:signing:cert:beta-signing": "fenix-repo",
                             "project:mobile:fenix:releng:signing:cert:production-signing": "fenix-repo",
+                            "project:mobile:fenix:releng:signing:cert:fennec-nightly-signing": "fenix-repo",
```
Comment (JohanLorenzo) [inline src/scriptworker/constants.py]: I just realized these `fenix` scopes were not guarded.

File: src/scriptworker/constants.py [hunk 2]
```diff
@@ -453,11 +464,12 @@
                     ),
                     "mobile": immutabledict(
                         {
+                            "android-components-repo": ("/mozilla-mobile/android-components",),
```
Comment (JohanLorenzo) [inline src/scriptworker/constants.py]: I just alphabetically ordered these lists.

File: src/scriptworker/constants.py [hunk 3]
```diff
@@ -453,11 +470,12 @@
...
                             "fenix-repo": ("/mozilla-mobile/fenix",),
+                            "fennec-profile-manager": ("/mozilla-mobile/fennec-profile-manager",),
```
Comment (MihaiTabara) [inline src/scriptworker/constants.py]: question: did we want a trailing `-repo` to be consistent with others or should we not bother given that this is going away once adhoc signing is up?

File: src/scriptworker/constants.py [hunk 4]
```diff
@@ -365,12 +376,18 @@
                             "project:mobile:fenix:releng:signing:cert:nightly-signing": "fenix-repo",
                             "project:mobile:fenix:releng:signing:cert:beta-signing": "fenix-repo",
                             "project:mobile:fenix:releng:signing:cert:production-signing": "fenix-repo",
+                            "project:mobile:fenix:releng:signing:cert:fennec-nightly-signing": "fenix-repo",
+                            # Fennec Beta and Fennec Release share the same key
+                            "project:mobile:fenix:releng:signing:cert:fennec-production-signing": "fenix-repo",
+                            "project:mobile:fennec-profile-manager:releng:signing:cert:fennec-nightly-signing": "fenix-repo",
```
Comment (MihaiTabara) [inline src/scriptworker/constants.py]: `s/fenix-repo/fennec-profile-manager`for this line and the next in their values? I'm assuming we want to be the value we defined at https://github.com/mozilla-releng/scriptworker/pull/433/files#diff-4635e904df4496f16271f8684ac489f5R475

File: src/scriptworker/constants.py (hunk 3 above)
Comment (JohanLorenzo) [reply src/scriptworker/constants.py]: Oops, you're right! Fixed!

File: src/scriptworker/constants.py (hunk 4 above)
Comment (JohanLorenzo) [reply src/scriptworker/constants.py]: 👍 Fixed

**Pull request merged**